lifecycle-build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+rm -f /home/vcap/app/.cloudfoundry/.placeholder
+rm -f /home/vcap/app/.profile.d/.placeholder
+cd /home/vcap || exit
```

Comment:
Er also, should this script have `set -e` at the top so any failed commands make it bust out with a nonzero exit code?